app/App.tsx

```diff
@@ -33,19 +33,19 @@ export const App = () => {
       ReactGA_4.initialize(config.GOOGLE_ANALYTICS_GA4_ID);
     }
 
-    return history.listen((loc) => {
```

Comment:
**note:** The effect hook will now re-run on location change so there's no need to listen to a history change.